src/main/java/mate/academy/lib/Injector.java

```diff
@@ -31,12 +31,12 @@ public static Injector getInstance(String mainPackageName) {
         return injector;
     }
 
-    public Object getInstance(Class<?> certainInterface) {
+    public static Object getInstance(Class<?> certainInterface) {
         Class<?> clazz = findClassExtendingInterface(certainInterface);
         return createInstance(clazz);
     }
```

Comment:
The method 'getInstance' is a bit misleading because it suggests that you can get an instance of any class, while it actually expects an interface that has an implementation marked with the @Dao annotation. Consider renaming this method to reflect that it's specifically for getting Dao implementations, such as 'getDaoInstance'.